package.json

```diff
@@ -70,11 +70,8 @@
     "react": "18.3.1",
     "react-dom": "18.3.1",
     "react-i18next": "14.1.3",
-    "react-quick-pinch-zoom": "5.1.0",
     "react-router-dom": "6.30.0",
     "recharts": "2.15.1",
-    "swiper": "11.2.5",
-    "use-resize-observer": "9.1.0",
```

Comment:
You removed dependencies but yarn.lock added 1000 lines
something's off

Reply:
oops its because I merged #2188 and thought removed lines were from vite deduplicating.

In reality we need https://github.com/oasisprotocol/rose-app/pull/307 and nevermind   https://github.com/oasisprotocol/rose-app/pull/307#issuecomment-3478469653

Reply:
Fixed in https://github.com/oasisprotocol/explorer/pull/2309

Reply:
thanks